build.savant

```diff
@@ -55,18 +56,18 @@ project(group: "com.inversoft.savant.plugin", name: "client-library", version: "
 }
 
 // Plugins
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.4")
-groovy = loadPlugin(id: "org.savantbuild.plugin:groovy:2.0.0-RC.4")
-groovyTestNG = loadPlugin(id: "org.savantbuild.plugin:groovy-testng:2.0.0-RC.5")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.4")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.4")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
+def groovy = loadPlugin(id: "org.savantbuild.plugin:groovy:2.0.0-RC.7")
+def groovyTestNG = loadPlugin(id: "org.savantbuild.plugin:groovy-testng:2.0.0-RC.7")
+def idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.7")
+def release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
 
 // Plugin settings
 groovy.settings.groovyVersion = "4.0"
-groovy.settings.javaVersion = "1.8"
+groovy.settings.javaVersion = "21"
```

Comment:
I would prefer to use Java 17. This is the version we are using across all of the savant plugins.

Reply:
Done, got to 17.